src/router/utils/HelperFunctions.jsx

```diff
@@ -50,3 +51,50 @@ export const removeFromWatchlist = async (video, videoDispatch) => {
     console.log(error);
   }
 };
+
```

Comment:
Suggestion: Instead of Console logging the error, showing it to the user would be better.

Reply:
Yes, I'm yet to add the toasts. Thank you!